diesel_cli/README.md

```diff
@@ -108,7 +108,7 @@ Runs the `down.sql` and then the `up.sql` for the most recent migration.
 ## `diesel print-schema`
 Prints table definitions for database schema.
 
-[pending-migrations]: https://docs.diesel.rs/diesel/migrations/fn.run_pending_migrations.html
+[pending-migrations]: https://docs.rs/migrations_internals/*/migrations_internals/fn.run_pending_migrations.html
```

Comment:
Oh, wow, we published that as "migrations-internals" and not "diesel-migrations-internals"? @sgrif, wanna change that?

Reply:
I think we need to `#[doc(inline)]` some stuff, that function should be exposed only in `diesel_migrations`

Reply:
It looks like we do have `#[doc(inline)]`, but that doesn't work for functions from other crates... Looks like we'll need to do some work there

Reply:
The correct link is now: https://docs.rs/diesel_migrations/*/diesel_migrations/fn.run_pending_migrations.html
It would be great to fix that before merging